### PR TITLE
Add iframe sample

### LINKED ISSUE #834: sample: binding to amp-iframe[src]

### PR DESCRIPTION
**Pull request by kul3r4**
Fixes #834

### CODE REVIEW

**Comment by kul3r4** [comment]
@ericlindley-g PTAL

File: src/20_Components/amp-bind.html [hunk 1]
```diff
@@ -118,17 +121,30 @@
     <p [class]="allAnimals[currentAnimal].style" class="greenBackground">Each animal has a different background color.</p>
     <!-- ## Binding Attribute Values -->
     <!--
-      Various AMP components support binding attribute values with `amp-bind`. For example, `amp-img` URLs can be changed.
+      Various AMP components support binding attribute values with `amp-bind` and here it follows
```
Comment (ericlindley-g) [inline src/20_Components/amp-bind.html]: One minor grammatical change:

"Various AMP components support binding attribute values with `amp-bind`. Below you can find a subset of examples"

(Note that as the list grows, we may want to have a simple list of links at the top to quickly jump to each sample)

File: src/20_Components/amp-bind.html [hunk 2]
```diff
@@ -118,17 +121,30 @@
...
+      a list of samples.
+      - `amp-img` URLs can be changed.
     -->
     <amp-img src="/img/Border_Collie.jpg" layout="responsive" width="300" height="200"
                                                                           [src]="allAnimals[currentAnimal].imageUrl">
     </amp-img>
 
     <!--
-      The src URL for videos embedded with `amp-video` can also be changed. At the moment, among all the AMP video components, only `amp-video` supports `amp-bind`.
+      - The `src` URL for videos embedded with `amp-video` can be changed. At the moment, among all the AMP video components, only `amp-video` supports `amp-bind`.
```
Comment (ericlindley-g) [inline src/20_Components/amp-bind.html]: I think amp-youtube now supports amp-bind (https://github.com/ampproject/amphtml/pull/8067)

File: src/20_Components/amp-bind.html (hunk 1 above)
Comment (kul3r4) [inline src/20_Components/amp-bind.html]: Good idea, I added links.

File: src/20_Components/amp-bind.html (hunk 2 above)
Comment (kul3r4) [inline src/20_Components/amp-bind.html]: Added!

**Pull request merged**